Makefile

```diff
@@ -70,7 +73,66 @@ run-kernel-gateway:
 			$(KG_IMAGE_NAME); \
 	fi;
 
-kill:
+###### tmpnb
+
+MAX_LOG_SIZE:=50m
+MAX_LOG_ROLLOVER:=10
+
+token-check:
+	@test -n "$(TOKEN)" || \
+		(echo "ERROR: TOKEN not defined (make help)"; exit 1)
```

Comment:
Good point.  I'll change it to `TMPNB_PROXY_AUTH_TOKEN` because I think we'll end up with others, like `TMPNB_API_AUTH_TOKEN`.